hedera-node/configuration/mainnet/upgrade/feeSchedules.json

```diff
@@ -1950,6 +1950,52 @@
           ]
         }
       },
+      {
+        "transactionFeeSchedule": {
+          "hederaFunctionality": "TokenCancelAirdrop",
+          "fees": [
+            {
+              "subType": "DEFAULT",
+              "nodedata": {
+                "constant": 7983519,
```

Comment:
Was thinking more about these feeSchedule changes, we can probably just add a constant and VPT. But if this works while validating tests, we can keep the same.

Reply:
For some reason, this doesn't work while testing. The cancel gets charged $1...what do you mean by `just add a constant and VPT`? Should all other configs in the feeSchedules.json file just be 0? I'm not sure if this will solve the problem.

I will debug the fee problems today, hopefully I will have time till the end of the day to figure it out.

Reply:
changed this and it works now